draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -337,35 +325,35 @@ HTTP features such as {{?HPACK=RFC7541}}.
 
 # Receiving RateLimit fields {#receiving-fields}
 
-A client MUST process the received `RateLimit` fields.
+A client MUST process the received RateLimit fields.
```

Comment:
What is the value of adding this requirement that is not enforceable?  Why MUST a client process these headers?  What does it actually mean to process them?

Reply:
A client conformant with this specification is expected to process the fields. If it doesn't, then it's not conformant to this specification (but can be conformant to HTTP). How would you state it?